### PR TITLE
move the content from README.md to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by steveortiz**

### CODE REVIEW

**Comment by steveortiz** [comment]
@rose-coste moving the content from README to CONTRIBUTING should help contributors find the contributing guidelines more easily... GitHub looks for this file and provides a link to those who fork it.  Will you please review and merge?

Also, I took out the part about not FORKING... I want to make it work for those who fork, and anyone not on our team can only contribute by forking.  I will get that done today.

**Comment by steveortiz** [comment]
@rose-coste - PR #428 should fix the forking issue we had previously.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+Find the directory for the product you are working on.
+
+Make any edits you may need directly through the Github website, through the Github desktop client, or on the command line.
+
+When you finish editing an article, you must commit the edit so Github can produce a pull request. In the Github website or Github desktop client, create a commit message following the guidelines below:
+
+- Provide a brief description of the change.
+- Use imperative mood. For example, "Add a paragraph about ..." vs "Added a paragraph about ...".
+- If you make a complex edit, explain why you are making the edit. For example, if you changing the formatting of an article because a   list should be ordered instead of unordered, say, "Switch list in middle of article to ordered so that there is clear progression of   steps".
```
Comment (meker12) [inline CONTRIBUTING.md]: For example, if you changing....  >>> 

For example, if you update an article to change list formatting, provide a commit message that explains the change: *Change bulleted list to numeric to show clear progression of steps*

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+  Note: Be sure that you are making commits to a branch. In the Github desktop client and the Github website, this can be achieved by   selecting the **Create a new branch for this commit and start a pull request** option below your commit message.
```
Comment (meker12) [inline CONTRIBUTING.md]: In the Github desktop client or website interface, you commit to a branch by selecting the  **Create a new branch for this commit and start a pull request** option below your commit message.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+Create a pull request for your branch.
+
+During the audit phase of the migration, after a pull request is made, you are responsible for merging the pull request.
```
Comment (meker12) [inline CONTRIBUTING.md]: Delete the ... "During the audit phase sentence"

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+Commit any new changes to the branch and merge your pull request once you have completed your edits.
```
Comment (meker12) [inline CONTRIBUTING.md]: Replace 29 with the following: 

After you complete any requested changes, commit them to the branch and merge your pull request.

**Comment by steveortiz** [comment]
@meker12 - I made the changes you suggested, and I also replaced Github with GitHub.

**Pull request merged**